features/fixed-periods.ethiopic.feature

```diff
@@ -4,30 +4,30 @@ Feature: Ethiopic Calendar fixed periods
 
     Scenario: Number of generated periods
         When a year is provided along a period
-            | year | periodType    | periodCount |
-            | 2015 | MONTHLY       | 12          |
-            | 2015 | DAILY         | 366         |
-            | 2014 | DAILY         | 365         |
-            | 2014 | WEEKLY        | 52          |
-            | 2015 | WEEKLY        | 52          |
-            | 2015 | BIWEEKLY      | 26          |
-            | 2015 | BIMONTHLY     | 6           |
-            | 2015 | QUARTERLY     | 4           |
-            | 2015 | SIXMONTHLY    | 2           |
-            | 2015 | SIXMONTHLYAPR | 2           |
-            | 2015 | YEARLY        | 10          |
-            | 2015 | FYNOV         | 10          |
+            | year | periodType      | periodCount |
+            | 2015 | Monthly         | 12          |
+            | 2015 | Daily           | 366         |
+            | 2014 | Daily           | 365         |
+            | 2014 | Weekly          | 52          |
+            | 2015 | Weekly          | 52          |
+            | 2015 | BiWeekly        | 26          |
+            | 2015 | BiMonthly       | 6           |
+            | 2015 | Quarterly       | 4           |
+            | 2015 | SixMonthly      | 2           |
+            | 2015 | SixMonthlyApril | 2           |
+            | 2015 | Yearly          | 54          |
+            | 2015 | FinancialNov    | 54          |
```

Comment:
I've reverted the changes and added an option so that apps can provide a custom set of `periodTypes`.
The library now exports two different sets of period types, there's a comment in the `period-types.ts` with additional information